.travis.yml

```diff
@@ -1,8 +1,32 @@
 language: scala
-jdk:
-- oraclejdk8
-- oraclejdk10
-script: sbt verify
+
+jobs:
+  include:
+    - stage: test
+      script: sbt verify
+      jdk: oraclejdk8
+
+    - script: sbt verify
+      jdk: oraclejdk10
+
+    - stage: publish
+      script: sbt ^publishSigned
```

Comment:
Given #247 I think this needs to selectively `^core/publishSigned ^plugin/publishSigned ...`

Reply:
Ahh, thanks for reminding me of that. I have disabled `genericTheme` publishing for sbt `0.13`. So that should solve it.